bindata/v4.1.0/kube-apiserver/localhost-recovery-sa.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: localhost-recovery-client
```

Comment:
any reason this is not a simple CVO manifest?

Reply:
> any reason this is not a simple CVO manifest?

CVO cannot remove resources.  It's merge logic is suspect (frequent failures from an old copy of library-go).  It has a 10 minute sync time.  And it doesn't logically manage this resource.

I would recommend using the CVO as little as possible.